app/views/account/users/_users_table.html.slim

```diff
@@ -0,0 +1,13 @@
+tbody
+  tr
+    td =link_to span= user.full_name, "#"
```

Comment:
formatting + create SHOW page and put link to user to show page here

Reply:
tip:
```
= link_to '#' do
  span= user.full_name
```